HOMEPAGE_README.md

```diff
@@ -0,0 +1,22 @@
+# REDI Sprint 2020 Frontend Development 2, Final Project
+
+Application Name: User account for a teenage-content portal
+
+Student Name: Stanka Svecova
+
+Description:
+A user can register to the portal and access user account with following functionalities:
+Update & delete personal information
+Create, post, edit & delete an article
+Upload a video-link (not the video itself)
+
+## Screenshots
+
+![screen shot](homepage_proposal.png)
```

Comment:
nice image please keep repo clean by placing this image into docs folder and deleting the existing screen shot there.

Reply:
there is now a pr placing the screenshot into the docs folder.